python/lsst/pipe/tasks/drpDiaCalculationPipe.py

```diff
@@ -0,0 +1,139 @@
+# This file is part of pipe_tasks.
+
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Pipeline for computing DiaObject summary/light curve values.
+"""
+
+__all__ = ["DrpDiaCalculationPipeTask",
+           "DrpDiaCalculationPipeConfig",
+           "DrpDiaCalculationPipeConnections"]
+
+import pandas as pd
+
+from lsst.meas.base import DiaObjectCalculationTask
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+
+
+class DrpDiaCalculationPipeConnections(pipeBase.PipelineTaskConnections,
+                                       dimensions=("tract", "patch", "skymap"),
```

Comment:
You pass in skymap here to the PipelinTaskConnections but then it's never used again; the other Input calls use only tract and patch. Is that intentional?

Reply:
The skymap here defines the tracts/patches. I think I want it here as if I did not and a repo has more than one skymap and therefore data with different tract/patch definitions, I wouldn't want them to grab tracts/patches with the same id numbers across different skymaps as they likely represent different areas of the sky.